lib/presentation/organisms/timer_organism.dart

```diff
@@ -24,6 +26,10 @@ class _TimerOrganismState extends State<TimerOrganism>
   @override
   void initState() {
     super.initState();
+    lockScreen = _prefs.getBool("isLockScreen") ?? lockScreen;
+    WakeLockController.instance.setWakeLock(lockScreen);
+    PlayerController.instance.setIsSound(!(_prefs.getBool("isSound") ?? true));
```

Comment:
Now with the new name there is no need to flip the value.
Remove the !

Reply:
Updated.